app/controllers/slot_ranges_controller.rb

```diff
@@ -18,13 +18,19 @@ def update
       @slot_range.slots.destroy_all
       @slot_range.update(slot_range_parameters)
     end
-    redirect_to edit_user_path(@guider), flash: { success: "Guider \"#{@guider.name}\" has been updated" }
+    redirect_to(
+      edit_user_path(@guider),
+      success: "Slot Range \"#{@slot_range.display_title}\" has been updated"
```

Comment:
Hmm... the thing is that schedules/slot ranges aren't actually editable. Once they're active, they've active. This is because they may have appointments allocated to their defined slots.

So the design was that they can create a new schedule any point from 6 weeks in advance. 

I suppose we could allow future schedules to be edited, but this feels like extra  logic at this stage?